scheduler/plugin/mock/wrappedplugin.go

```diff
@@ -0,0 +1,262 @@
+	mock *MockNormalizeScorePluginExtender
+}
+
+// NewMockNormalizeScorePluginExtender creates a new mock instance
+func NewMockNormalizeScorePluginExtender(ctrl *gomock.Controller) *MockNormalizeScorePluginExtender {
+	mock := &MockNormalizeScorePluginExtender{ctrl: ctrl}
+	mock.recorder = &MockNormalizeScorePluginExtenderMockRecorder{mock}
+	return mock
+}
+
+// EXPECT returns an object that allows the caller to indicate expected use
+func (m *MockNormalizeScorePluginExtender) EXPECT() *MockNormalizeScorePluginExtenderMockRecorder {
+	return m.recorder
+}
+
+// BeforeNormalizeScore mocks base method
+func (m *MockNormalizeScorePluginExtender) BeforeNormalizeScore(ctx context.Context, state *framework.CycleState, pod *v1.Pod, scores framework.NodeScoreList) *framework.Status {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "BeforeNormalizeScore", ctx, state, pod, scores)
+	ret0, _ := ret[0].(*framework.Status)
+	return ret0
+}
+
+// BeforeNormalizeScore indicates an expected call of BeforeNormalizeScore
+func (mr *MockNormalizeScorePluginExtenderMockRecorder) BeforeNormalizeScore(ctx, state, pod, scores interface{}) *gomock.Call {
+	mr.mock.ctrl.T.Helper()
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "BeforeNormalizeScore", reflect.TypeOf((*MockNormalizeScorePluginExtender)(nil).BeforeNormalizeScore), ctx, state, pod, scores)
+}
+
+// AfterNormalizeScore mocks base method
+func (m *MockNormalizeScorePluginExtender) AfterNormalizeScore(ctx context.Context, state *framework.CycleState, pod *v1.Pod, scores framework.NodeScoreList, normalizeScoreResult *framework.Status) *framework.Status {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "AfterNormalizeScore", ctx, state, pod, scores, normalizeScoreResult)
+	ret0, _ := ret[0].(*framework.Status)
+	return ret0
+}
+
+// AfterNormalizeScore indicates an expected call of AfterNormalizeScore
+func (mr *MockNormalizeScorePluginExtenderMockRecorder) AfterNormalizeScore(ctx, state, pod, scores, normalizeScoreResult interface{}) *gomock.Call {
+	mr.mock.ctrl.T.Helper()
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "AfterNormalizeScore", reflect.TypeOf((*MockNormalizeScorePluginExtender)(nil).AfterNormalizeScore), ctx, state, pod, scores, normalizeScoreResult)
+}
+
+// MockOption is a mock of Option interface
+type MockOption struct {
+	ctrl     *gomock.Controller
+	recorder *MockOptionMockRecorder
+}
+
+// MockOptionMockRecorder is the mock recorder for MockOption
+type MockOptionMockRecorder struct {
+	mock *MockOption
+}
+
+// NewMockOption creates a new mock instance
+func NewMockOption(ctrl *gomock.Controller) *MockOption {
+	mock := &MockOption{ctrl: ctrl}
+	mock.recorder = &MockOptionMockRecorder{mock}
+	return mock
+}
+
+// EXPECT returns an object that allows the caller to indicate expected use
+func (m *MockOption) EXPECT() *MockOptionMockRecorder {
+	return m.recorder
+}
+
+// apply mocks base method
+func (m *MockOption) apply(arg0 *options) {
```

Comment:
I run 
```
//go:generate mockgen -destination=./mock/$GOFILE -source=$GOFILE store, filterPluginExtender, scorePluginExtender, normalizeScorePluginExtender
```
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/128/files#diff-88e5e83973b677bd5a4adc7d8450d0981ba9d1a6a69db4f84bbc67ee6bdb2492R12

I think my implementation way is not good. Could you give me some advice?

Reply:
Maybe resolve.  Fix it tonight.